packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -12,11 +12,11 @@ annotations:
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: rancher-cis-benchmark
 apiVersion: v1
-appVersion: v2.0.4
+appVersion: v2.0.5
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
 keywords:
 - security
 name: rancher-cis-benchmark
-version: 2.0.4
+version: 2.0.5
```

Comment:
We should bump the chart to rc version before the release. We should 2.0.5-rc1 until we are ready for release

Reply:
updated to rc version